trio/_core/_tests/test_asyncgen.py

```diff
@@ -182,7 +182,7 @@ async def async_main() -> None:
         assert record == []
 
     _core.run(async_main)
-    assert record == ["innermost"] + list(range(100))
+    assert record == ["innermost", *list(range(100))]
```

Comment:
I don't think the `list(...)` is necessary.

Reply:
Fixed in 4f7621d